rholang/src/main/scala/coop/rchain/rholang/interpreter/compiler/IndexMapChain.scala

```diff
@@ -13,14 +15,11 @@ final case class IndexMapChain[T](chain: Vector[DeBruijnIndexMap[T]]) {
 
   def get(name: String): Option[IndexContext[T]] = chain.head.get(name)
 
-  def find(name: String): Option[(IndexContext[T], Int)] = {
-    import cats.implicits.catsStdInstancesForVector
-    import cats.Foldable.ops.toAllFoldableOps
+  def find(name: String): Option[(IndexContext[T], Int)] =
     chain.zipWithIndex.collectFirstSome {
```

Comment:
@JosDenmark well, I didn't expect errors for these imports. ;)
```
[error] .../compiler/IndexMapChain.scala:19:11: object ops in object Foldable is deprecated (since 2.2.0): Use cats.syntax object imports
[error]     chain.zipWithIndex.collectFirstSome {
[error]           ^
```

Reply:
😆